src/bootstrap/sanity.rs

```diff
@@ -86,6 +86,10 @@ pub fn check(build: &mut Build) {
         build.config.nodejs = Some(node);
     } else if let Some(node) = have_cmd("nodejs".as_ref()) {
         build.config.nodejs = Some(node);
+    } else if let Some(node) = have_cmd("node.exe".as_ref()) {
```

Comment:
No need for those extra checks, `have_cmd` will automatically consider `.exe` as well for each command.